images/quay-upload/Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM registry.access.redhat.com/ubi8/ubi-minimal:8.6
```

Comment:
This image is not used for operating the service, but for developing the service. Therefore it should be moved to `test/images` for the moment. Hopefully we can refactor the repo in the future to improve the layout.

Reply:
Done.